### PR TITLE
#CX-9184 : Accessibility issues in new UX for EP/Address Book 

### PR DESCRIPTION
**Pull request by rituparnadey1807**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/tabs/Tabs.ts [hunk 1]
```diff
@@ -646,6 +646,7 @@ export namespace Tabs {
 
       const key = event.code;
       const { shiftKey } = event;
+      console.log("More Menu Tab Visible--", this.isMoreTabMenuVisible);
```
Comment (prakhar-tiwari) [inline web-components/src/components/tabs/Tabs.ts:649]: Remove this console.log

File: web-components/src/mixins/FocusTrapMixin.ts [hunk 2]
```diff
@@ -276,6 +278,7 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
 
     protected setFocusableElements() {
       this.focusableElements = this.findFocusable(this.shadowRoot!, new Set());
+      console.log("focusableElements$", this.focusableElements);
```
Comment (prakhar-tiwari) [inline web-components/src/mixins/FocusTrapMixin.ts:281]: Also here

**Review by prakhar-tiwari — APPROVED** [review]

File: web-components/src/mixins/FocusTrapMixin.ts [hunk 3]
```diff
@@ -364,11 +366,14 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
       this.addEventListener("keydown", this.handleKeydownFocusTrap);
       this.addEventListener("focus-visible", this.handleFocusVisible as EventListener);
       const self = this;
-      document.addEventListener("on-component-update", () => {
+      document.addEventListener("on-widget-update", () => {
```
Comment (vikcool2pr) [inline web-components/src/mixins/FocusTrapMixin.ts:369]: please don't change event name, and where "on-widget-update" & "deactivate-focus-trap" is dispatched?

**Review by vikcool2pr — CHANGES_REQUESTED** [review]

File: web-components/src/mixins/FocusTrapMixin.ts (hunk 3 above)
Comment (rituparnadey1807) [reply web-components/src/mixins/FocusTrapMixin.ts:369]: This name change was done by Parag, on-widget-update is dispatched from desktop(TransferModalContext.tsx -> on menu overlay open). @parsoman

**Comment by rituparnadey1807** [comment]
BUILD

**Comment by rituparnadey1807** [comment]
BUILD

**Review by vikcool2pr — APPROVED** [review]

**Review by vikcool2pr — DISMISSED** [review]

**Review by prakhar-tiwari — DISMISSED** [review]

**Comment by rituparnadey1807** [comment]
BUILD

**Comment by rituparnadey1807** [comment]
BUILD

**Comment by rituparnadey1807** [comment]
BUILD

**Review by vikcool2pr — APPROVED** [review]

**Review by prakhar-tiwari — APPROVED** [review]

**Comment by rituparnadey1807** [comment]
BUILD

**Pull request merged**